### PR TITLE
update /desktop to vbt

### LINKED ISSUE #1654: /desktop to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /desktop to vbt
* made the ‘BENEFITS FOR ALL INDUSTRY SECTORS’ animations much much simpler

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-overview.png) and the [demo](http://www.ubuntu.com-1654-desktop-overview.demo.haus/desktop)

## Issue / Card

Fixes #1654

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Please can the animation be inverted, to match the live site.

Thanks

**Comment by pmahnke** [comment]
@spencerbygraves updated animation

**Comment by yaili** [comment]
Design review:
- [ ] The images in "Looks great on the latest devices" and "Available on a huge range of hardware" should move to another line in S screen, right now they're getting squeezed next to the text
- [ ] The icons in "BENEFITS FOR ALL INDUSTRY SECTORS" are too close to each other in S screen, could they become a 2x2 grid?

File: templates/desktop/index.html [hunk 1]
```diff
@@ -1,140 +1,149 @@
...
-            <h1>Ubuntu for desktops</h1>
-            <p class="intro">Learn how the Ubuntu desktop operating system powers millions of PCs and laptops around the world.</p>
-            <p>
-                <a href="/download/desktop" class="button--primary">Download Ubuntu</a>
-            </p>
-            <p>
-                <a href="/desktop/features">Take a look&nbsp;&rsaquo;</a>
-            </p>
-        </div>
+{% block content %}{# 0 -180px #}
```
Comment (barrymcgee) [inline templates/desktop/index.html]: What does this comment refer to?

**Comment by pmahnke** [comment]
@yaili added them to a 2x2 grid

speed up the transition, happy to remove if you want

**Comment by yaili** [comment]
- [ ] I still the image in "Available on a huge range of hardware" on S screen
screen shot 2017-06-13 at 09 55 01

- [ ] I think we should remove the animation because it's a bit random, and we have the text underline on hover

**Comment by anthonydillon** [comment]
I have a fix for this upstream here:
https://github.com/vanilla-framework/vanilla-framework/pull/1111.

**Comment by pmahnke** [comment]
@yaili @anthonydillon I fixed this locally

**Comment by barrymcgee** [comment]
Demo: http://www.ubuntu.com-1654-desktop-overview.demo.haus/

**Comment by pmahnke** [comment]
@spencerbygraves (@yaili) removed the hover effect and that image at S... demo updated

**Pull request merged**